elpy.el

```diff
@@ -593,9 +593,13 @@ virtualenv.
     (add-hook 'xref-backend-functions #'elpy--xref-backend nil t))
   (cond
    (elpy-mode
-    (elpy-modules-buffer-init))
+    (elpy-modules-buffer-init)
+    (add-hook 'pre-command-hook 'elpy-autodoc--pre-command nil t)
+    (add-hook 'post-command-hook 'elpy-autodoc--post-command nil t))
    ((not elpy-mode)
-    (elpy-modules-buffer-stop))))
+    (elpy-modules-buffer-stop)
+    (remove-hook 'pre-command-hook 'elpy-autodoc--pre-command t)
+    (remove-hook 'post-command-hook 'elpy-autodoc--post-command t))))
```

Comment:
Can't this be a module? Then this init/deinit can be moved to a module definition, and people can deactivate it entirely when they want.